src/main/java/core/basesyntax/service/impl/ProductDaoImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
```

Comment:
this class is actually placed in an incorrect package. Let's move it to the 'impl' package of dao package